railties/lib/rails/test_unit/runner.rb

```diff
@@ -59,6 +57,27 @@ def compose_filter(runnable, filter)
         end
 
         private
+          def test_paths(argv)
+            filtered_test_paths(argv) || default_test_paths
+          end
+
+          def default_test_paths
+            Rake::FileList[default_test_glob].exclude(default_test_exclude_glob)
+          end
+
+          def default_test_glob
+            ENV.fetch("DEFAULT_TEST", DEFAULT_TEST_GLOB)
+          end
+
+          def default_test_exclude_glob
+            ENV.fetch("DEFAULT_TEST_EXCLUDE", DEFAULT_TEST_EXCLUDE_GLOB)
```

Comment:
This is a public class so the constants will be added to the public API. Don't think we want that. Since we've already extracted to a method, why don't we inline them and use standard `[] ||` as well?